tests/TableTestStateTest.php

```diff
@@ -26,6 +26,28 @@ public function testInitialization()
         $this->assertEquals($originRecords, $state);
     }
 
+    public function testInitializationViaPrepareTableTestState()
+    {
+        $datasetMock = collect($this->getJsonFixture('initialization/dataset.json'));
+        $this->mockGettingDataset($datasetMock);
+
+        $mock = $this
+            ->getMockBuilder(TestCase::class)
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $reflection = new ReflectionClass($mock);
+
+        $reflection->getMethod('setGlobalExportMode')->invoke($mock);
+
+        $prepareTableTestState = $reflection->getMethod('prepareTableTestState')->invoke($mock, 'test_models');
```

Comment:
let's move it to the separate method which will return reflection of the TestCase class with enabled global export mode